cdci_data_analysis/analysis/instrument.py

```diff
@@ -186,6 +186,7 @@ def read_frontend_files(self,
                 sentry_client.capture('raven.events.Message',
                                            message=f'Error while setting input scw_list file from the frontend {e}')
             raise RequestNotUnderstood("Error while setting input scw_list from the frontend")
+        self.set_pars_from_dic(par_dic, verbose=verbose)
```

Comment:
Is this somthing that appeared to be necessary after the other PR? I am perplexed why it passed tests?

Reply:
because we never tested the range of certain input values directly within the dispatcher-app, but in the dispatcher-plugin-integral

eg [https://github.com/oda-hub/dispatcher-plugin-integral/blob/2889b5d7d3fa0feba7740dbf628e25abb1d74c56/tests/test_isgri_plugin.py#L425-L459](https://github.com/oda-hub/dispatcher-plugin-integral/blob/2889b5d7d3fa0feba7740dbf628e25abb1d74c56/tests/test_isgri_plugin.py#L425-L459)

Reply:
ok, can it be also tested directly in dispatcher-app?

Reply:
yes it should, it was never done before